[BUG]

**Describe the bug**
The Release tag workflow is failing:

internal/process/esm_loader.js:74
    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_UNKNOWN_BUILTIN_MODULE]: No such built-in module: node:timers/promises
    at new NodeError (internal/errors.js:322:[7](https://github.com/Githubguy132010/Arch-Linux-without-the-beeps/actions/runs/13353050774/job/37292006808#step:5:8))
    at Loader.builtinStrategy (internal/modules/esm/translators.js:2[8](https://github.com/Githubguy132010/Arch-Linux-without-the-beeps/actions/runs/13353050774/job/37292006808#step:5:9)9:11)
    at new ModuleJob (internal/modules/esm/module_job.js:63:26)
    at Loader.getModuleJob (internal/modules/esm/loader.js:258:11)
    at async ModuleWrap.<anonymous> (internal/modules/esm/module_job.js:78:21)
    at async Promise.all (index 0)
    at async link (internal/modules/esm/module_job.js:83:[9](https://github.com/Githubguy132010/Arch-Linux-without-the-beeps/actions/runs/13353050774/job/37292006808#step:5:10)) {
  code: 'ERR_UNKNOWN_BUILTIN_MODULE'
}
Error: Process completed with exit code 1.

**Solution:**

Ensure that all Git configurations are properly unset and no orphan processes are left running. You can update the workflow file .github/workflows/automated-tagging-versioning.yaml to include better cleanup steps.

--- a/.github/workflows/automated-tagging-versioning.yaml
+++ b/.github/workflows/automated-tagging-versioning.yaml
@@ -18,10 +18,19 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: '14'
+          node-version: '18'
 
       - name: Install Semantic Release
         run: npm install -g semantic-release @semantic-release/changelog @semantic-release/git @semantic-release/github
+
+      - name: Unset Git Configurations
+        run: |
+          git config --global --unset-all user.name || true
+          git config --global --unset-all user.email || true
+
+      - name: Kill Orphan Processes
+        run: |
+          pkill -f node || true
 
       - name: Automated Tagging and Versioning
         env:

